### ISSUE TITLE
RadSideDrawer navigation broke android application: 

### ISSUE THREAD

**Issue by gbonline** [NativeScript/NativeScript#8299]
Hi, I'm developing and android app ng based, focused on a mobile device with Android 8.1 API 27, I 've the Android SDK updated so I've an update emulator AVD machine with Android 8.1-API 27. The avd have 2GB of ram, and in my case it have a custom hardware profile, but I think it doesn't matter. The app : NS 6.3.3, tns.core 6.3.2, tns-android 6.3.1

My app was created from the sidedrawer template and I've some issue during  navigation between pages  from sidedrawer links
I've tried an empty project builded on this template, and it have showed too the issue: the app suddenly crash without any exception info.
You can try it with 

```
tns create mysidedrawer --template tns-template-drawer-navigation-ng

tns run android --emulator

```

then repeat 

> - open sidedrawer
> - click on a link

clicking about ten or more times, it's variable, the app crash

**Comment by vahidvdn** [comment]
The crash is related to #7867 and [here ](https://github.com/NativeScript/NativeScript/issues/7867#issuecomment-544647538) is the temporary workaround.

In this case, you can close the issue.

**Comment by gbonline** [comment]
I applied the change

 and it seems to have worked.

Thank you

**Comment by vahidvdn** [comment]
@gbonline It seems this is fixed with #8320
